README.md

```diff
@@ -0,0 +1,20 @@
+# Dark Triad Classification Project Overview
+
+Available June 16th, 2023
+
+## TL;DR
+
+**Goal:**
+
+- Predict score category for dark triad traits based on SD3 questionnaire.
+- Provide user-interface that allows for people to take SD3 questionnaire and view actual scores vs. predicted scores by trait-type.
+
+**Languages/Applications Used:**
+
+- Python (EDA, data visualization, machine learning, user-interface, api-creation)
+- GitHub (project organization, version control)
+
+**Team Overview:**
+
+- **Team Size:** 4
+- **Project Development Time:** 9 days
```

Comment:
I would recommend adding our names.
Do we need a table of content?

Reply:
I'll add the names.

The table of contents was something I was going to add a little later since it may change.